src/Microsoft.ML.Core/Utilities/DoubleParser.cs

```diff
@@ -555,6 +562,7 @@ private static bool TryParseCore(ReadOnlySpan<char> span, ref int ich, ref bool
                     break;
 
                 case '.':
+                case ',':
                     goto LPoint;
```

Comment:
So with all this infrastructure to set the decimal marker, are we permitting both `.` and `,` to be treated as a decimal marker?